### DISCUSSION TITLE
Livewire

### DISCUSSION THREAD

**Discussion by jhagan-91** [nWidart/laravel-modules#1397] [Q&A] [answered]
I'm just trying to establish are the livewire components included within the modules structure?

For whatever reason it's not clear to me, and I haven't tested this package yet to definitively answer this myself.

Thanks in advance.

**Comment by dcblogdev** [comment, marked as answer]
Hi, no they are not but livewire does work with modules install https://github.com/mhmiton/laravel-modules-livewire to add Livewire commands for Modules.
